job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -65,7 +66,7 @@ pipeline{
                 build job: 'nightly/7.59.x', propagate: false, wait: true, parameters: [
                         [\$class: 'StringParameterValue', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-rhba-7.12/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '712'],
-                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '7.12.0'],
+                        [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: "${PRODUCT_VERSION}"],
```

Comment:
Is this correct? UMB_VERSION should match as well.

Reply:
I see, UMB_VERSION is correct, but PRODUCT_VERSION should not be used in this case.

Reply:
good catch